storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -143,7 +155,21 @@ The following options are available when using |gcs|:
 - --google-access-key = <ACCESS KEY ID>
 - --google-secret-key = <SECRET ACCESS KEY>
 - --google-bucket = <BUCKET NAME>
+- --google-storage-class=name
+
+.. note::
 
+    The Google storage class name options are the following:
+    
+    * STANDARD
+    * NEARLINE
+    * COLDLINE
+    * ARCHIVE
+    
+    .. seealso::
+    
+     `Google storage classes <https://cloud.google.com/storage/docs/storage-classes>`_
+    
```

Comment:
also, mention in the case of google. Default storage class depends on the storage class of bucket.
**https://cloud.google.com/storage/docs/changing-default-storage-class**

Reply:
Added

Reply:
Default keyword is missing

Reply:
Fixed.